fintech/src/main/java/ru/kslacker/fintech/service/CityWeatherServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package ru.kslacker.fintech.service;
+
+import lombok.experimental.ExtensionMethod;
+import org.springframework.stereotype.Service;
+import ru.kslacker.fintech.dataaccess.api.CityRepository;
+import ru.kslacker.fintech.dataaccess.api.WeatherRepository;
+import ru.kslacker.fintech.dto.CreateCityDto;
+import ru.kslacker.fintech.dto.CreateWeatherDto;
+import ru.kslacker.fintech.dto.WeatherDto;
+import ru.kslacker.fintech.mapping.WeatherMapper;
+import ru.kslacker.fintech.models.City;
+import ru.kslacker.fintech.models.Weather;
+import ru.kslacker.fintech.service.api.CityWeatherService;
+import ru.kslacker.fintech.validation.service.api.ValidationService;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@ExtensionMethod({WeatherMapper.class})
```

Comment:
Лучше не использовать, даже в документации Lombok  написано
`Current status: hold - Currently we feel this feature will not move out of experimental status anytime soon, but it will not significantly change and support for it is unlikely to be removed in future versions of lombok either.`